helm/templates/client/clusterrole.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ols-proxy-backend-register
```

Comment:
this is the role that, when granted to a user/service account, allows that account to make OLS api requests, right?

let's give it a better name like `ols-user` or `ols-requester`